unit_tests/test_events.py

```diff
@@ -421,19 +421,6 @@ def test_default_filters(self):
                                "10.12.0.219: std::runtime_error (view update generator not plugged to push updates)") \
                 .publish()
 
-            DatabaseLogEvent.RUNTIME_ERROR() \
```

Comment:
cause this logic was introduced, ontop of the filter:
```
if sct_config.get('cluster_backend').startswith("k8s"):
```

since this log line is k8s setup only, we don't need to filter on any other setup.
 
yes it's not directly related, and also would be introduced by #7041